snmp/datadog_checks/snmp/snmp.py

```diff
@@ -55,7 +61,9 @@ def __init__(self, *args, **kwargs):
 
         self.ignore_nonincreasing_oid = is_affirmative(self.init_config.get('ignore_nonincreasing_oid', False))
 
-        self.profiles = self.init_config.get('profiles', {})  # type: Dict[str, Dict[str, Any]]
+        self.profiles = self.init_config.get('profiles')  # type: Dict[str, Dict[str, Any]]
+        if self.profiles is None:
+            self.profiles = get_default_profiles()
```

Comment:
This has become invalid type-wise; updated suggestion:

```suggestion
        profiles = self.init_config.get('profiles')
        if profiles is None:
            profiles = get_default_profiles()

        self.profiles = profiles  # type: Dict[str, Dict[str, Any]]
```